packages/wrangler/src/__tests__/standard-pricing.test.ts

```diff
@@ -166,6 +176,36 @@ describe("standard-pricing", () => {
 		}
 	`);
 	});
+
+	it("should warn user about new pricing if enabled and usage_model specified after deprecation date", async () => {
+		jest.setSystemTime(new Date(2024, 2, 2));
+
+		msw.use(...mswSuccessDeploymentScriptMetadata);
+		writeWranglerToml({ usage_model: "bundled" });
+		writeWorkerSource();
+		mockSubDomainRequest();
+		mockUploadWorkerRequest();
+
+		mockStandardEnabled(true);
+
+		await runWrangler("deploy ./index");
+
+		expect(std).toMatchInlineSnapshot(`
```

Comment:
Can you make this `std.out` to make the snapshot a bit cleaner?

Reply:
Updated to use `std.warn`, `std.out` doesn't contain the warning.